src/Uno.UI/Controls/CalendarDatePickerValueChangedEventArgs.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class CalendarDatePickerDateChangedEventArgs
```

Comment:
This modification should be in another PR as it not related to the MenuFlyout.

Reply:
Yes, I know - I did not know that any changes in my clone of Uno would be automatically replicated to previously created PR. And I don't know how to reverse this (but probably you, Uno Masters, can remove these two changes from PR).